nepalingo-web/src/components/DailyQuiz.tsx

```diff
@@ -1,32 +1,46 @@
 import React from "react";
-import { useNavigate } from "react-router-dom"; // Import useNavigate
+import { useNavigate } from "react-router-dom";
 import Button from "./Button";
-const DailyQuiz: React.FC = () => {
-  const backgroundImageUrl =
-    "https://t3.ftcdn.net/jpg/00/73/08/22/360_F_73082224_ay4Tus31QNHNmGSIty53ZE6mBrBc47cV.jpg"; // Set your image URL
-  const navigate = useNavigate(); // Get the navigate function
+
+interface DailyQuizProps {
+  backgroundImageUrl: string;
+  quizYourselfText: string;
+  descriptionText: string;
+  buttonText: string;
+}
+
+const DailyQuiz: React.FC<DailyQuizProps> = ({
+  backgroundImageUrl,
+  quizYourselfText,
+  descriptionText,
+  buttonText,
+}) => {
+  const navigate = useNavigate();
 
   const handleStartQuizClick = () => {
-    // Redirect to /flashcard when the button is clicked
     navigate("/flashcard");
   };
 
   return (
     <div
-      className="daily-quiz-card bg-cover text-white p-8 rounded-lg"
-      style={{ backgroundImage: `url(${backgroundImageUrl})` }}
```

Comment:
Donot directly use a background image add an image which covers the whole height and width and also add an overlay on top of it with some color like bg-black/90